tests/sdk/v2/driver/test_client.py

```diff
@@ -322,18 +333,29 @@ def test_sets_auth(
             def test_unauthorized(endpoint_mocker, client: DriverClient):
                 endpoint_mocker(
                     # Specified in:
-                    # https://github.com/zapatacomputing/workflow-driver/blob/2b353476d5b0161da31584533be208611a131bdc/openapi/src/resources/workflow-definition.yaml#L26
+                    # https://github.com/zapatacomputing/workflow-driver/blob/main/openapi/src/resources/workflow-definitions.yaml#L25
```

Comment:
**suggestion**: can we use permalinks instead of branches? It's possible to go from a commit permalink to the latest revision, but it's much more difficult in the opposite order. The diff between `commit...main` also allows us to see if schema was changed since the time we wrote the client.